### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by GraceAbike**
## In this project, I've set up a "Hello world" repository.
 image

- Create a new GitHub repo.
- Create a new branch according to the GitHub flow rules.
- Add a `.gitignore` file.
- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
- Add a descriptive README file.
- Add 3 files (each one in a separate commit):
- A .html file that includes a header with the text "Hello Microverse!". Use the `h1` tag for that.
- A .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in - your GitHub repository.

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```
Comment (Reem-lab) [inline .hintrc:18]: - [ ] Great on set up the linters files but I think to install the packages for hintrc and stylelint if you scroll down [here](https://github.com/microverseinc/linters-config/tree/master/html-css) you will see the steps please add it to your project ✅ and please don't forget after installing the packages to put `node_modules` in gitignore file 🚀🚀
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @GraceAbike , 👋🏻👋🏻 

Good job so far! 🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file. ✅

## Required Changes ♻️
_Check the comments under the review._
- [ ] I think you forget to create a test.md file as required so kindly create it and write the required paragraph inside it. 
image


 - [ ] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc (hunk 1 above)
Comment (GraceAbike) [reply .hintrc:18]: @Reem-lab 
Thank you for reviewing the project
I set up all requirements I think If I merge everything is going to come together
image

**Review by muneebulrehman — APPROVED** [review]
Hi @GraceAbike , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- Proper linter setup
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**